### PR TITLE
Renaming several files

### PR DESCRIPTION
**Pull request by marcelomorgado**
Refs #36

### CODE REVIEW

File: packages/tasit-sdk/src/ethers.test.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		const oldValue = await contract.getValue();
+		const newValue = `I like cats ${randomWallet.mnemonic}`;
+
+		const tx = await contract.setValue(newValue);
+
+		await waitForEvent("ValueChanged", [wallet.address, oldValue, newValue]);
+		await ropstenProvider.waitForTransaction(tx.hash);
+	});
+
+	it("should broadcast signed tx", async function() {
```
Comment (pcowgill) [inline packages/tasit-sdk/src/ethers.test.js]: Does this one need an xit too? If not, why not?

File: packages/tasit-sdk/src/tasitSdk.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Account } from "./index";
```
Comment (pcowgill) [inline packages/tasit-sdk/src/tasitSdk.test.js]: For some reason my gut reaction is that I would have done
`import { Account } from "./tasitSdk";`
even though the current approach is actually more directly testing what the user would actually be importing, which makes for an arguably better test. @marcelomorgado what do you think?

File: packages/tasit-sdk/src/ethers.test.js (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/ethers.test.js]: Sorry, xits are there by mistake. Do you think that is better they are xits by default?

File: packages/tasit-sdk/src/tasitSdk.test.js (hunk 2 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/tasitSdk.test.js]: I agree with you, I've fixed it on PR https://github.com/tasitlabs/TasitSDK/blob/c1df5e13930835f64c8c5e37a91d2a4b5e58836e/packages/tasit-sdk/src/tasitSdk.test.js#L1

Comment (marcelomorgado) [reply packages/tasit-sdk/src/tasitSdk.test.js]: I can fix it on this branch either. You call.

File: packages/tasit-sdk/src/ethers.test.js (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-sdk/src/ethers.test.js]: I think until we move to ganache and make them run faster, having timeouts enabled for the test suite and keeping these as xit's is the way to go.

File: packages/tasit-sdk/src/tasitSdk.test.js (hunk 2 above)
Comment (pcowgill) [reply packages/tasit-sdk/src/tasitSdk.test.js]: It being changed in the other PR is fine with me! Thanks.

File: packages/tasit-sdk/src/ethers.test.js (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/ethers.test.js]: Done

**Pull request merged**